lib/src/clojure_lsp/feature/refactor.clj

```diff
@@ -43,6 +43,9 @@
 (defmethod refactor :cycle-fn-literal [{:keys [loc]}]
   (r.transform/cycle-fn-literal loc))
 
+(defmethod refactor :promote-fn [{:keys [loc uri db]}]
```

Comment:
I feel this should allow the function name to be passed in.

Reply:
Easy enough. Good suggestion.